data/mods/Magiclysm/materials.json

```diff
@@ -69,6 +69,34 @@
       { "fuel": 1, "smoke": 1, "burn": 1, "volume_per_turn": "100 ml" }
     ]
   },
+  {
+    "type": "material",
+    "id": "parabolan_wool",
+    "name": "Parabolan Wool",
+    "density": 5,
+    "specific_heat_liquid": 0.02,
+    "specific_heat_solid": 0.02,
+    "soft": true,
+    "latent_heat": -50,
```

Comment:
Negative?

Reply:
I was mostly testing to see what it would do to clothing warmth levels.